packages/ui/v2/core/layouts/SettingsLayout.tsx

```diff
@@ -319,7 +319,9 @@ export default function SettingsLayout({
     <Shell
       flexChildrenContainer
       {...rest}
-      SidebarContainer={<SettingsSidebarContainer className="hidden lg:flex" />}
+      SidebarContainer={
+        <SettingsSidebarContainer className="absolute inset-y-0 z-50 m-0 hidden h-screen translate-x-0 transform overflow-y-scroll border-gray-100 bg-gray-50 transition duration-200 ease-in-out lg:flex" />
```

Comment:
this adds so many classes and I don't know why we need them. can you explain?

Reply:
Hey @PeerRich,  I added these classes to side bar to have the exact style of the others, as shown in this comment https://github.com/calcom/cal.com/issues/4505

Reply:
Hey @PeerRich, I added these classes to side bar to have the exact style of the others, as shown in this comment https://github.com/calcom/cal.com/issues/4505, but when removing the changes I realized that these changes are no longer necessary
Loom video from main branch: https://www.loom.com/share/b0edde6090804c8ebd55f0a93d819e65